common/namespace/handler.go

```diff
@@ -576,7 +576,7 @@ func (d *HandlerImpl) UpdateNamespace(
 			configVersion++
 		}
 
-		if configurationChanged || activeClusterChanged {
+		if (configurationChanged || activeClusterChanged) && isGlobalNamespace {
```

Comment:
Can we only check if activeClusterChanged

Reply:
maybe - I didn't write the original code, so I want to make this change relatively small.

Reply:
see comment below, it really should not be this complicated.